### PR TITLE
[Admin] Fix typos.

### PR DESCRIPTION
**Pull request by kbrandl**
Changes **Office Store** -> **AppSource** throughout the docs. Note that I've intentionally *not* updated `ms.date` in any of these articles, since none of the changes are substantive.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[0b6e848](https://github.com/OfficeDev/office-js-docs-pr/pull/1182/commits/0b6e848e7c9166a457c7c76e8ed1000c7482c8d8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-change-OfficeStore-to-AppSource/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | |
[docs/overview/schemas/content/OfficeAppBasicTypesV1_0.xsd](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-change-OfficeStore-to-AppSource/docs/overview/schemas/content/OfficeAppBasicTypesV1_0.xsd) | :white_check_mark:Succeeded | |
[docs/overview/schemas/content/OfficeAppManifestV1_1.xsd](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-change-OfficeStore-to-AppSource/docs/overview/schemas/content/OfficeAppManifestV1_1.xsd) | :white_check_mark:Succeeded | |
[docs/overview/schemas/mail/OfficeAppBasicTypesV1_0.xsd](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-change-OfficeStore-to-AppSource/docs/overview/schemas/mail/OfficeAppBasicTypesV1_0.xsd) | :white_check_mark:Succeeded | |
[docs/overview/schemas/mail/OfficeAppManifestV1_1.xsd](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-change-OfficeStore-to-AppSource/docs/overview/schemas/mail/OfficeAppManifestV1_1.xsd) | :white_check_mark:Succeeded | |
[docs/overview/schemas/taskpane/OfficeAppBasicTypesV1_0.xsd](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-change-OfficeStore-to-AppSource/docs/overview/schemas/taskpane/OfficeAppBasicTypesV1_0.xsd) | :white_check_mark:Succeeded | |
[docs/overview/schemas/taskpane/OfficeAppManifestV1_1.xsd](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-change-OfficeStore-to-AppSource/docs/overview/schemas/taskpane/OfficeAppManifestV1_1.xsd) | :white_check_mark:Succeeded | |
[docs/reference/manifest/alternateid.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-change-OfficeStore-to-AppSource/docs/reference/manifest/alternateid.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/iconurl.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-change-OfficeStore-to-AppSource/docs/reference/manifest/iconurl.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/scopes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-change-OfficeStore-to-AppSource/docs/reference/manifest/scopes.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907172250216807-1182%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=AneWyU9B4JwnRN3KgaHgwQSWt08C2bXT3LQM7iC5Pig%3D&st=2019-07-17T22%3A46%3A20Z&se=2019-08-17T22%3A51%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/overview/explore-with-script-lab.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ localization_priority: Normal
 
 # Explore Office JavaScript API using Script Lab
 
-The [Script Lab add-in](https://store.office.com/app.aspx?assetid=WA104380862), which is available free from the Office store, enables you to explore the Office JavaScript API while you're working in an Office program such as Excel or Word. Script Lab is a convenient tool to add to your development toolkit as you prototype and verify functionality you want in your add-in.
+The [Script Lab add-in](https://appsource.microsoft.com/en-us/product/office/WA104380862), which is available free from AppSource, enables you to explore the Office JavaScript API while you're working in an Office program such as Excel or Word. Script Lab is a convenient tool to add to your development toolkit as you prototype and verify functionality you want in your add-in.
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/explore-with-script-lab.md]:
```suggestion
The [Script Lab add-in](https://appsource.microsoft.com/product/office/WA104380862), which is available free from AppSource, enables you to explore the Office JavaScript API while you're working in an Office program such as Excel or Word. Script Lab is a convenient tool to add to your development toolkit as you prototype and verify functionality you want in your add-in.
```

File: docs/overview/explore-with-script-lab.md [hunk 2]
```diff
@@ -57,14 +57,14 @@ Script Lab is supported for Excel, Word, and PowerPoint on the following clients
 
 ## Next steps
 
-To use Script Lab in Excel, Word, or PowerPoint, install the [Script Lab add-in](https://store.office.com/app.aspx?assetid=WA104380862) from the Office Store. 
+To use Script Lab in Excel, Word, or PowerPoint, install the [Script Lab add-in](https://appsource.microsoft.com/en-us/product/office/WA104380862) from AppSource.
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/explore-with-script-lab.md]:
```suggestion
To use Script Lab in Excel, Word, or PowerPoint, install the [Script Lab add-in](https://appsource.microsoft.com/product/office/WA104380862) from AppSource. 
```

File: docs/overview/explore-with-script-lab.md [hunk 3]
```diff
@@ -57,14 +57,14 @@ Script Lab is supported for Excel, Word, and PowerPoint on the following clients
...
+To use Script Lab in Excel, Word, or PowerPoint, install the [Script Lab add-in](https://appsource.microsoft.com/en-us/product/office/WA104380862) from AppSource. 
 
 You're welcome to expand the sample library in Script Lab by contributing new snippets to the [office-js-snippets](https://github.com/OfficeDev/office-js-snippets#office-js-snippets) GitHub repository.
 
 When you're ready to create your first Office Add-in, try out the quick start for [Excel](../quickstarts/excel-quickstart-jquery.md), [Outlook](/outlook/add-ins/quick-start?context=office/dev/add-ins/context), [Word](../quickstarts/word-quickstart.md), [OneNote](../quickstarts/onenote-quickstart.md), [PowerPoint](../quickstarts/powerpoint-quickstart.md), or [Project](../quickstarts/project-quickstart.md).
 
 ## See also
 
-- [Get Script Lab](https://store.office.com/app.aspx?assetid=WA104380862)
+- [Get Script Lab](https://appsource.microsoft.com/en-us/product/office/WA104380862)
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/explore-with-script-lab.md]:
```suggestion
- [Get Script Lab](https://appsource.microsoft.com/product/office/WA104380862)
```

**Comment by kbrandl** [comment]
@ElizabethSamuel-MSFT thanks for the feedback! I inadvertently merged this PR prior to incorporating your suggestions (to remove locale from the 3 links); I'll incorp those changes via another PR momentarily. Thanks!

**Pull request merged**